src/test/java/com/browserstack/steps/SetupSteps.java

```diff
@@ -0,0 +1,127 @@
+package com.browserstack.steps;
+
+import com.browserstack.ParallelTest;
+import com.browserstack.local.Local;
+import com.browserstack.util.OsUtility;
+import com.browserstack.util.Utility;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.Scenario;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.remote.CapabilityType;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.io.FileReader;
+import java.net.URL;
+import java.nio.file.Path;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+public class SetupSteps {
+
+    private final StepData stepData;
+    private Local local;
+    protected String chromeDriverBaseLocation = Path.of(System.getProperty("user.dir"), "/src/test/resources/chromeDriver").toString();
+
+    private static final String PASSED = "passed";
+    private static final String FAILED = "failed";
+    private static final String URL = "https://bstackdemo.com";
+    private static final String WEBDRIVER_CHROME_DRIVER = "webdriver.chrome.driver";
+
+    public SetupSteps(StepData stepData) {
+        this.stepData = stepData;
+    }
+
+    @Before
+    public void setUp(Scenario scenario) throws Exception {
+        JSONObject config;
+        JSONObject capabilityObject;
```

Comment:
please name the json object w.r.t the data they possess or the function they perform in the code.

Reply:
done